Author: Steffen Moeller <moeller@debian.org>
Description: This test contacts a localhost, which fails, like all networking should fail during testing? Unsure.
Index: htslib/test/test.pl
===================================================================
--- htslib.orig/test/test.pl
+++ htslib/test/test.pl
@@ -57,7 +57,7 @@ test_command($opts,cmd=>'test-bcf-transl
 test_convert_padded_header($opts);
 test_rebgzip($opts);
 test_logging($opts);
-test_plugin_loading($opts);
+#test_plugin_loading($opts);
 test_realn($opts);
 
 print "\nNumber of tests:\n";
